test/junit/scala/tools/nsc/interpreter/CompletionTest.scala

```diff
@@ -231,6 +231,14 @@ class CompletionTest {
     checkExact(completer, s"($ident: Int) => tia")(ident)
   }
 
+  @Test
+  def completionWithComment(): Unit = {
+    val intp = newIMain()
+    val completer = new PresentationCompilerCompleter(intp)
+    assert(completer.complete("Array(1, 2, 3).map(_ + 1) /* then we do reverse */ .rev").candidates.contains("reverseMap"))
```

Comment:
It may be more clear if you would extract the supposed inputs as variables, and use a triple-quote-block with `.stripMargin`.

Reply:
Extracted into variables, I agree it looks better. Thanks for the tip!